ticTacToe/README.md

```diff
@@ -0,0 +1,12 @@
+## Tired of Global Thermonuclear War? Play Tic Tac Toe!
+
```

Comment:
I like how your README file is not only informative but also shows your personality!